_versions/1.7/guides/topic/continuum.adoc

```diff
@@ -6,202 +6,214 @@ include::../attributes.adoc[]
 }
 ----
 
 For each call you are intending to do, add a method and use annotations to describe the behavior.
-
-Then, in your resource, just use the `CountriesService`:
+You can combine the REST Client with the https://quarkus.io/guides/smallrye-fault-tolerance[fault tolerance extension] to handle failure gracefully.
+Then, in your resource, just use the `ElementService` interface:
 
 [source, java]
 ----
-@Path("/country")
-public class CountriesResource {
+@Path("/elem")
+public class ElementResource {
 
-    @Inject
     @RestClient
-    CountriesService countriesService;
+    ElementService elements;
 
     @GET
-    @Path("/name/{name}")
-    @Produces(MediaType.APPLICATION_JSON)
-    public Set<Country> name(@PathParam("name") String name) {
-        return countriesService.getByName(name);
+    @Path("/{position}")
+    public Element name(@PathParam("position") int position) {
+        return elements.getElement(position);
     }
 }
 ----
 
 But you may be wondering where the URL is configured as it's not in the code.
-Remember, it must not be hard-coded because the url likely depends on the environment.
+Remember, it must not be hard-coded because the URL likely depends on the environment.
 The URL is configured in the application configuration:
 
 [source, text]
 ----
-# the format is interface-name/mp-rest/url=the-service-url
-org.acme.restclient.CountriesService/mp-rest/url=https://restcountries.eu/rest
+element-service/mp-rest/url=http://localhost:9001
 ----
 
-The url can now be updated during the deployment or at launch time using system properties or environment variables.
+The URL can now be updated during the deployment or at launch time using system properties or environment variables.
+
+Quarkus is not limited to HTTP.
+You can use https://quarkus.io/guides/grpc-getting-started[gRPC] or https://quarkus.io/guides/smallrye-graphql[GraphQL], two prominent players in the microservice space. 
 
 == Being reactive
 
-Application requirements have changed drastically over the last few years.
-For any application to succeed in the era of cloud computing, big data or IoT, going reactive is increasingly becoming the architecture style to follow.
+Application requirements have changed drastically over the last few years. 
+For any application to succeed in the era of cloud computing, big data, or IoT, going reactive is increasingly becoming the architecture style to follow.
+
+Today’s users embrace applications with milliseconds of response time, 100% uptime, lower latency, push data instead of pull, higher throughput, and elasticity. However, these features are nearly impossible to achieve by using yesterday’s software architecture without a considerable investment in resources, infrastructure, and tooling. The world changed, and having dozens of servers, long response times (> 500 ms), downtime due to maintenance or waterfalls of failures does not meet the expected user experience.
```

Comment:
```suggestion
Today’s users embrace applications with milliseconds of response time, 100% uptime, lower latency, push data instead of pull, higher throughput, and elasticity. However, these features are nearly impossible to achieve using yesterday’s software architecture without a considerable investment in resources, infrastructure, and tooling. The world changed, and having dozens of servers, long response times (> 500 ms), downtime due to maintenance or waterfalls of failures does not meet the expected user experience.
```